### ISSUE TITLE
Real model - MEMC model comparison

### ISSUE THREAD

**Issue by bpbond** [Microbial-Explicit-Model/MEMC#38]
Have you guys talked about running a comparison between (e.g.) the real MEND model and the version implemented in MEMC? It seems absolutely crucial to do for at least one or two models--demonstrating that the MEMC dynamics look like, at least, those of the real model.

Also, we'll want language in the abstract and conclusion noting that MEMC models are not the full/real thing, and intent of MEMC is for rapid exploratory work and hypothesis testing...right?

**Comment by jianqiuz** [comment]
Yes. There are two major comparison goals I have: 1) different combinations of kinetic formulations 2) different initial pool assignments.  After some testing, KD and I have decided #2 is too much work. So here in MEMC 1, we mainly focus on #1. Different models use different pool structures and kinetics, making it difficult to directly compare hypotheses behind kinetic formulations. MEMC 1 allows such comparison of kinetic formulations under a single pool structure, not to replace the full model.

**Comment by bpbond** [comment]
Thanks @jianqiuz . Right, different kinetic formulations seems like a great focus for this initial documentation paper; but still, I think we'll need at least one figure documenting that the abstracted combination of kinetics we're calling (for example) "MEND" produces behavior consistent with the original model.

@kdorheim and I are pulling together data for this exercise -- see #50
